tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1185,7 +1185,7 @@ def test_large_generation(self):
         )
 
         generated_ids = model.generate(
-            input_features, do_sample=False, max_length=20, language="<|en|>", task="transcribe"
+            input_features, do_sample=False, max_length=20, language="en", task="transcribe"
```

Comment:
test implicity :man_shrugging:

Reply:
Feel free to add a fast test to check for this - along the lines of https://github.com/huggingface/transformers/blob/57ffd8ab4c833e26b2288769f6031f94870a102c/tests/models/whisper/test_modeling_whisper.py#L406
But without the fp16 or beam search, and instead just doing the generation with `language="en"`, then `language="<en>"`, then `langauge="English"`, something like:

```python
def test_generate_language(self):
        config, input_dict = self.model_tester.prepare_config_and_inputs()
        input_features = input_dict["input_features"]
        model = WhisperForConditionalGeneration(config).to(torch_device)

        # test language code
        model.generate(input_features, language="en")
        # test tokenizer code
        model.generate(input_features, language="<|en|>")
        # test language name
        model.generate(input_features, language="English")
```